src/workerd/jsg/modules.h

```diff
@@ -518,6 +524,16 @@ class ModuleRegistryImpl final: public ModuleRegistry {
         js.v8Error(kj::str("No such module \"", specifier.toString(), "\".")));
   }
 
+  Value resolveInternalImport(jsg::Lock& js, const kj::StringPtr specifier) override {
+    auto specifierPath = kj::Path(specifier);
```

Comment:
Just something to keep in mind for later... I'll soon be working on changes to treat import specifiers as proper URLs rather than just kinda-sorta-paths.